src/app/(nextjs_migration)/(authenticated)/user/welcome/details/page.tsx

```diff
@@ -0,0 +1,75 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import Script from "next/script";
+
+import { getL10n } from "../../../../../functions/server/l10n";
+import { getLocale } from "../../../../../../utils/fluent.js";
+import { authOptions } from "../../../../../api/utils/auth";
+
+export async function generateMetadata() {
+  const l10n = getL10n();
+  return {
+    title: "Welcome - Details", // FIXME l10n.getString("breach-meta-title"),
+    twitter: {
+      card: "summary_large_image",
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      images: ["/images/og-image.webp"],
+    },
+    openGraph: {
+      title: l10n.getString("brand-fx-monitor"),
+      description: l10n.getString("meta-desc-2"),
+      siteName: l10n.getString("brand-fx-monitor"),
+      type: "website",
+      url: process.env.SERVER_URL,
+      images: ["/images/og-image.webp"],
+    },
+  };
+}
+
+export default async function UserBreaches() {
+  const l10n = getL10n();
+
+  return (
+    <>
+      <main>
+        <h1>Enter the details you want to protect</h1>
+        <p>
+          We’ll use this to find exposures of your personal information, and
+          then guide you step-by-step on how to fix it.
+          <a>Why do we need this info?</a>
+        </p>
+
+        <section>
+          <form method="post" action="scanning">
+            <div>
+              <label for="name">First name: </label>
```

Comment:
I think all these `for=` attributes are wrong.

```
Error: The value of the for attribute of the label element must be the ID of a non-hidden form control.

[From line 14, column 15; to line 14, column 32](https://validator.w3.org/nu/#l14c32)

          <label for="name">First
```

```suggestion
              <label for="firstname">First name: </label>
```

Reply:
Yeah, sorry I wasn't expecting anyone to jump on it when it was in draft state. No worries though, it's super helpful to have eyes on things as early as possible, but I don't want you to waste your time on stuff that's not quite ready either.